<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_end">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>35</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>42.121.4.45</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>35</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>结束IP：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>开始IP：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>21,22,3389,80</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_portBegin">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_portEnd">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>进度：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_port">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>61</y>
        <width>681</width>
        <height>281</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_begin">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>9</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>42.121.4.41</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>701</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>网卡列表</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>611</width>
         <height>181</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_interface">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>20</y>
         <width>31</width>
         <height>181</height>
        </rect>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>691</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>IP查询</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>IP地址或者域名：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>331</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>www.cnsecer.com</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>20</y>
         <width>31</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>441</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="action_portScan"/>
    <addaction name="action_host"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_portScan"/>
   <addaction name="action_host"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="action_portScan">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/port.png</normaloff>:/images/images/port.png</iconset>
   </property>
   <property name="text">
    <string>端口扫描</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/about.png</normaloff>:/images/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_host">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/ftp.png</normaloff>:/images/images/ftp.png</iconset>
   </property>
   <property name="text">
    <string>网卡信息</string>
   </property>
   <property name="toolTip">
    <string>网卡信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_portBegin</tabstop>
  <tabstop>pushButton_portEnd</tabstop>
  <tabstop>textEdit_port</tabstop>
  <tabstop>lineEdit_begin</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_interface</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_end</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit_port</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
